.github/SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Reporting Security Issues
+
+If you believe you have found a security vulnerability in SWR, we encourage you to let us know right away.
```

Comment:
```suggestion
If you believe you have found a security vulnerability in the AI SDK, we encourage you to let us know right away.
```

Reply:
my bad, fixed 😄

Reply:
I've also added code of conduct file if you'll have a minute or so 😄 

https://github.com/vercel/ai/pull/672